zipkin-common/src/main/scala/com/twitter/zipkin/test/FunSuiteWithJUnit.scala

```diff
@@ -0,0 +1,9 @@
+package com.twitter.zipkin.test
+
+import org.scalatest.junit.JUnitRunner
+import org.junit.runner.RunWith
+import org.scalatest.FunSuite
+
+@RunWith(classOf[JUnitRunner])
+class FunSuiteWithJUnit extends FunSuite {
```

Comment:
adding a common dep here is not so hot imho, as we already have a lot of coupling. Can you just add RunWith to all the tests? Then, in the main build file, put a TODO to replace boilerplate `@RunWith(classOf[JUnitRunner])` with getting scalatest plugin working?